src/lib/Cards/GameCard.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import type { Game } from '$lib/games';
+	import CardLabel from '$lib/shared/Label/CardLabel.svelte';
+	import OfflineLabel from '$lib/shared/Label/OfflineLabel.svelte';
+	import { formatTargetGroups } from '$lib/Utils/formatTargetGroups';
+
+	export let game: Game;
+
+	const targetGroupString = formatTargetGroups(game.targetGroup);
+</script>
+
+<article>
+	<h1>{game.name}</h1>
+	{#if game.offline}
+		<OfflineLabel />
+	{/if}
+	<ul>
+		<li><CardLabel label={targetGroupString} icon={undefined} /></li>
+		<li><CardLabel label={game.category} icon={undefined} /></li>
+		<li><CardLabel label={`Min. ${game.minimumPlayers} `} icon="group" /></li>
+	</ul>
+</article>
+
+<style>
+	article {
+		background-color: var(--color-white);
+		border-radius: 1.5em;
+		padding: 1em;
+		justify-content: space-between;
+		box-shadow: 0px 2px 4px 2px rgba(21, 45, 21, 0.1);
+		/* filter: drop-shadow(0 0.2em 0.2em hsl(92, 30%, 64%)) alternatief shadow */
```

Comment:
```suggestion
```

Reply:
Dit is die 'andere' drop shadow van jou, wil je die niet nog uitproberen?

Reply:
Jaa, was wel wat leuker, maar ik heb liever zo min mogelijk (onnodige) comments in dingen die live staan. Maar laat voor nu maar staan. Kunnen we altijd nog later naar kijekn.